delivery_auto_refresh/data/ir_config_parameter.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2018 Tecnativa - Pedro M. Baeza
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo noupdate="1">
+    <record id="set_default_carrier" model="ir.config_parameter">
```

Comment:
Let's improve the design to set it via settings using a field with the `config_parameter` parameter. For example: https://github.com/odoo/odoo/blob/15.0/addons/base_setup/models/res_config_settings.py#L16

Reply:
I'm working on it